first_hour_on_terra/README.md

```diff
@@ -0,0 +1,10 @@
+# First Hour on Terra
+
+## Overview
+
+The notebooks in this directory show how to perform a number of tasks using Terra. They are intended to give a new Terra user an overview of helpful references and best practices that can be built upon.
+
+## Getting started
+
+Run the [working_with_data_collections.ipynb](https://github.com/todo) notebook to learn about creating data collections and adding them to your workspaces.
```

Comment:
Fix the todo link

Reply:
Yes, once this PR merged in to the main branch of this repo, I will be able to replace that link with the actual URL.

Reply:
We can figure out that the URL will be https://github.com/DataBiosphere/terra-axon-examples/blob/main/first_hour_on_terra/working_with_data_collections.ipynb . So you can put that in if you wanted to do sooner than later.

Reply:
updated!